### PR TITLE
PRIME-1672/1673 Show Renewal Alert Only Within 90 Days + Text Updates for HA Org Info  and Paper Enrolment

### PR DESCRIPTION
**Pull request by mtpultz**

### CODE REVIEW

File: prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/administrators-page/administrators-page.component.html [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <app-page [busy]="busy">
-  <app-page-header>PharmaNet Site Registration</app-page-header>
+  <app-page-header>PharmaNet Health Authority Information</app-page-header>
```
Comment (mtpultz) [inline prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/administrators-page/administrators-page.component.html:2]: Apparently we don't want to see the words "Site Registration" anywhere in Health Authorities.

**Review by AnaisHebert — DISMISSED** [review]
🦝lgtm

**Review by Paahn — APPROVED** [review]
yahooo

**Review by AnaisHebert — APPROVED** [review]
duck

**Pull request merged**